models/alerts/re_data_alerts.sql

```diff
@@ -4,8 +4,10 @@ select
     message,
     last_value_text as value,
     time_window_end
-from
-    {{ ref('re_data_anomalies') }}
+from {{ ref('re_data_anomalies') }}
+qualify row_number() over (partition by model order by time_window_end desc) = 1
```

Comment:
I don't think that all databases support `qualify` for example Redshift doesn't, and for PostgreSQL simple `limit 1` will be more efficient.
What I want to say we'll need a database specific macro to safely implement this logic

Reply:
There actually exists such a [macro](https://docs.getre.io/master/docs/reference/macros/data_filtering#filter_remove_duplicates) on redata used for data filtering. Good opportunity to use a public macro internally 😄.

Also @ugemir , I think what you want to do is get the latest status for each (model, test_name) pair from the tests history.